crates/api_common/src/community.rs

```diff
@@ -10,6 +10,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Deserialize)]
 pub struct GetCommunity {
   pub id: Option<CommunityId>,
+  /// Example: star_trek , or star_trek@xyz.tld
   pub name: Option<String>,
```

Comment:
@Nutomic The problem is that for `GetCommunity` and `GetPersonDetails`, we need to be able to optionally fetch by a shortname which includes the domain. 

This provides that by parsing a shortname to an `actor_id` and reading that from the DB. It doesn't aggressively fetch that federated item if its not in the DB, because you wouldn't see it in the front end in the first place.